src/briefcase/config.py

```diff
@@ -419,7 +419,7 @@ def __init__(
             )
 
     def __repr__(self):
-        return f"<{self.bundle}.{self.app_name} v{self.version} AppConfig>"
+        return f"<{self.bundle_identifier} ({self.app_name}) v{self.version} AppConfig>"
```

Comment:
Is there a particular reason you've added the app name here? The bundle identifier should be unique enough for diagnostic purposes, which is what the `repr` is being used for.

Reply:
Got it, I wasn't sure if this was for diagnostic purpose or for repl cases. The only reason to include it would be to reduce confusion caused by the automatic conversion of the bundle identifier by making it clear that yes, the app name is still used elsewhere.

I'll make the change shortly to remove the `app_name` from the repr.